go.mod

```diff
@@ -1,48 +1,48 @@
 module github.com/getsops/sops/v3
 
-go 1.23.0
+go 1.24.0
```

Comment:
@getsops/maintainers same question as https://github.com/getsops/sops/pull/1938#discussion_r2330672433 and https://github.com/getsops/sops/pull/1941#discussion_r2338482230.